### PR TITLE
Enable `stylecheck` linter in test files

### LINKED ISSUE #2962: Enable linters for test packages
**Is your feature request related to a problem? Please describe.**
As of now linters are disabled from test packages in .golangci.yaml file:
```
issues:
  exclude-rules:
    - path: '_test.go'
      linters:
        - errcheck                      # Errors may be ignored in tests.
        - unparam                       # Tests might have unused function parameters.
        - lll
        - dupl
        - misspell
        - nestif
        - gci
```
**Describe the solution you'd like**
After enabling this linter, fix all kind of linter issues coming up in test packages.

### PR DESCRIPTION
**Pull request by anishbista60**
## Change Overview

Enables the linter `stylecheck`. See #2962.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [x] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test
- [x] :building_construction: Build

## Issues

- #3034
- #2962

## Test Plan

- [ ] :muscle: Manual
- [x] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

**Comment by anishbista60** [comment]
@julio-lopez works fine in my machine.

**Comment by anishbista60** [comment]
#3034

**Comment by anishbista60** [comment]
looks fine in my local machine

**Comment by pavannd1** [comment]
@anishbista60 could you please add some description above to explain why the linter has been removed? I see you have linked another PR here but it's unclear why these changes are required.

**Comment by anishbista60** [comment]
> @anishbista60 could you please add some description above to explain why the linter has been removed? I see you have linked another PR here but it's unclear why these changes are required.

This is PR is solving issue `2962`. Instead of enable entire linter in single PR , i created the different PR for each llinter  on the recommondation of @julio-lopez . That linking was done mistakely . if i mentioned `2962` in every single PR. Then we have to re opne that issue again and again . SO, that i mentioned this PR at `3034`

**Comment by anishbista60** [comment]
@pavannd1 sorry it is actually beinng enabled. mistake in description

File: pkg/controllers/repositoryserver/repositoryserver_controller_test.go [hunk 1]
```diff
@@ -46,7 +46,6 @@ import (
 	"github.com/kanisterio/kanister/pkg/poll"
 	"github.com/kanisterio/kanister/pkg/resource"
 	"github.com/kanisterio/kanister/pkg/secrets"
-	"github.com/kanisterio/kanister/pkg/secrets/repositoryserver"
```
Comment (julio-lopez) [inline pkg/controllers/repositoryserver/repositoryserver_controller_test.go:49]: it seems that the import without an alias would be preferred, even if it results in a larger change
@hairyhum @pavannd1 @viveksinghggits thoughts?

File: pkg/secrets/secrets_test.go [hunk 2]
```diff
@@ -23,7 +23,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	secerrors "github.com/kanisterio/kanister/pkg/secrets/errors"
-	"github.com/kanisterio/kanister/pkg/secrets/repositoryserver"
```
Comment (julio-lopez) [inline pkg/secrets/secrets_test.go:26]: ditto

**Review by julio-lopez — CHANGES_REQUESTED** [review]

**Comment by anishbista60** [comment]
@julio-lopez sorry yet to fix some error.

File: pkg/secrets/secrets_test.go (hunk 2 above)
Comment (anishbista60) [reply pkg/secrets/secrets_test.go:26]: ok

File: pkg/controllers/repositoryserver/repositoryserver_controller_test.go (hunk 1 above)
Comment (anishbista60) [reply pkg/controllers/repositoryserver/repositoryserver_controller_test.go:49]: ok

Comment (viveksinghggits) [reply pkg/controllers/repositoryserver/repositoryserver_controller_test.go:49]: Yes, let's use import without alias.

**Comment by anishbista60** [comment]
@julio-lopez looks fine in my machine now . Let's merge it .

**Review by julio-lopez — APPROVED** [review]

**Comment by julio-lopez** [comment]
🥇Thanks for doing this.

**Pull request merged**